src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.search-bar--large {
+  height: 70px;
+  padding-left: 26px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar--large:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar--large:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar--small {
+  height: 42px;
+  padding-left: 13px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar--small:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input-field {
+  font-family: "Avenir Book", sans-serif;
+  color: #3d4e61;
+  border: none;
+  outline: none;
+  padding: 0;
+  width: 100%;
+}
+
+.search-bar--large__input-field {
+  height: 19px;
+  margin-left: 17px;
+  font-size: 16px;
+}
+
+.search-bar--small__input-field {
+  height: 19px;
+  margin-left: 9px;
+  font-size: 14px;
+}
+
+::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar--large__input-field:focus {
+  font-family: "Avenir Heavy", sans-serif;
+}
+
+.search-bar--small__input-field:focus {
+  font-family: "Avenir Heavy", sans-serif;
+}
```

Comment:
seems like this code is duplicated.
why do you set font-family on focus but not by default?

Reply:
because that's the font-family that should be in focus according to the layout